### DISCUSSION TITLE
Facebook get post method

### DISCUSSION THREAD

**Discussion by Manohar7781** [obsei/obsei#330] [Ideas]
The new update from pyfacebook documentation tells that to get post information we need to use get info method. https://github.com/sns-sdks/python-facebook/blob/master/examples/get_facebook_post_info.py 

Correct me If I am wrong

**Comment by lalitpagaria** [comment]
I think this API doesn't give Post Comments. For comment we need this API https://github.com/sns-sdks/python-facebook/blob/master/pyfacebook/api/facebook/common_edges/comments.py
